x-pack/plugins/monitoring/public/alerts/lib/security_toasts.tsx

```diff
@@ -127,11 +57,5 @@ export const showSecurityToast = (alertingHealth: AlertingFrameworkHealth) => {
     return;
   }
 
-  if (!isSufficientlySecure && !hasPermanentEncryptionKey) {
-    showTlsAndEncryptionError();
-  } else if (!isSufficientlySecure) {
-    showTlsError();
-  } else if (!hasPermanentEncryptionKey) {
-    showEncryptionError();
-  }
+  showTlsAndEncryptionError();
```

Comment:
I think we should still check for these cases:

```suggestion
if (!isSufficientlySecure || !hasPermanentEncryptionKey) {
  showTlsAndEncryptionError();
}
```